go.sum

```diff
@@ -290,9 +290,11 @@ github.com/oklog/run v1.0.0/go.mod h1:dlhp/R75TPv97u0XWUtDeV/lRKWPKSdTuV0TZvrmrQ
 github.com/onsi/ginkgo v1.6.0/go.mod h1:lLunBs/Ym6LB5Z9jYTR76FiuTmxDTDusOGeTQH+WWjE=
 github.com/onsi/ginkgo v1.7.0 h1:WSHQ+IS43OoUrWtD1/bbclrwK8TTH5hzp+umCiuxHgs=
 github.com/onsi/ginkgo v1.7.0/go.mod h1:lLunBs/Ym6LB5Z9jYTR76FiuTmxDTDusOGeTQH+WWjE=
+github.com/onsi/ginkgo v1.8.0 h1:VkHVNpR4iVnU8XQR6DBm8BqYjN7CRzw+xKUbVVbbW9w=
```

Comment:
Do you know why does this file change? You didn't change any dependencies so it should not :thinking:

Reply:
That was a mistake, it was most likely updated via my go plugin in VS Code.  I'll back it out.

Reply:
This file would have changed when I merged the changes from "Switch to rabit-hole 2.0.0" commit https://github.com/terraform-providers/terraform-provider-rabbitmq/commit/5470b5ca8630e95a45b84660a861106c26843d34

Would it make more sense to close the PR and rebase on my master and do a new PR to this master?